queries/rates/utils.ts

```diff
@@ -64,6 +65,44 @@ export const mapLaggedDailyPrices = (prices: Candles): Prices => {
 	});
 };
 
+const markets = [
+	Synths.sETH,
+	Synths.sBTC,
+	Synths.sLINK,
+	Synths.sSOL,
+	Synths.sAVAX,
+	Synths.sMATIC,
+	Synths.sAAVE,
+	Synths.sUNI,
+	Synths.sEUR,
+	'sXAU',
+	'sXAG',
+	'sWTI',
+] as const;
+
+const map: Record<typeof markets[number], string> = {
+	[Synths.sETH]: 'ethereum',
+	[Synths.sBTC]: 'bitcoin',
+	[Synths.sLINK]: 'chainlink',
+	[Synths.sSOL]: 'solana',
+	[Synths.sAVAX]: 'avalanche-2',
+	[Synths.sMATIC]: 'matic-network',
+	[Synths.sAAVE]: 'aave',
+	[Synths.sUNI]: 'uniswap',
+	[Synths.sEUR]: 'euro',
+	sXAU: '',
+	sXAG: '',
+	sWTI: '',
```

Comment:
we might want to add APE and DYDX over here - right?

Reply:
Yeah this could also be part of ticket #828, or at least from what I see is before merging this ticket need to check in that there's no conflicts between this PR and 828.

Noticed that the utils.ts from the MarketDetails is removed within this PR, while #828 might introduce it again.